bot/cogs/utils.py

```diff
@@ -40,6 +42,20 @@
 ICON_URL = "https://www.python.org/static/opengraph-icon-200x200.png"
 
 
+def get_pep_zero_embed() -> Embed:
+    """Send information about PEP 0."""
+    pep_embed = Embed(
+        title=f"**PEP 0 - Index of Python Enhancement Proposals (PEPs)**",
+        description=f"[Link](https://www.python.org/dev/peps/)"
+    )
+    pep_embed.set_thumbnail(url=ICON_URL)
+    pep_embed.add_field(name="Status", value="Active")
+    pep_embed.add_field(name="Created", value="13-Jul-2000")
+    pep_embed.add_field(name="Type", value="Informational")
+
+    return pep_embed
```

Comment:
This should still stay as a static method inside of the class instead of being defined outside as it's only useful inside of that class.